backend/dataall/modules/dataset_sharing/services/share_managers/lf_share_manager.py

```diff
@@ -519,3 +514,13 @@ def glue_client(self):
             region=self.target_environment.region,
             database=self.shared_db_name,
         )
+
+    def clean_up_share(self) -> bool:
+        """"
+        1) deletes deprecated shared db in target account
+        Returns
+        -------
+        True if clean-up succeeds
+        """
+        self.delete_shared_database()
+        return True
```

Comment:
Maybe we can skip this function completely and directly call delete_shared_database in `dataall/modules/dataset_sharing/services/data_sharing_service.py:210`

Reply:
Good idea - changed to directly call  `self.delete_shared_database()`